### ISSUE TITLE
abnormal luteolysis

### ISSUE THREAD

**Issue by anna-anagnostop** [mgijax/mammalian-phenotype-ontology#3616]
Add new term:

abnormal luteolysis (a child of abnormal corpus luteum morphology MP:0002679; abnormal ovary development MP:0003582; and abnormal ovulation cycle MP:0009344)
DEF: any anomaly in the structural and functional degradation of the corpus luteum occurring at the end of the luteal phase of both the estrous and menstrual cycles in the absence of pregnancy; during normal luteolysis, loss of the capacity to synthesize and secrete progesterone (functional luteolysis) is followed by loss of the cells that comprise the corpus luteum (structural luteolysis)
SOURCE: https://en.wiktionary.org/wiki/luteolysis, GO:0001554 
SYN: abnormal CL regression 
SYN: abnormal corpus luteum regression
SYN: abnormal luteal regression
Comment: The regression of the corpus luteum is essential to reset the ovarian cycle, so that mammals obtain another chance to become pregnant.

Term placement based on [GO:0001554 ](http://www.informatics.jax.org/vocab/gene_ontology/GO:0001554)

**Comment by anna-anagnostop** [comment]
Added new term:

abnormal luteolysis MP:0031423 under 3 parent terms along with comment and synonyms. No EQ added, as luteolysis GO:0001554 needs to be imported first.

**Comment by anna-anagnostop** [comment]
Remove it from abnormal corpus luteum morphology MP:0002679; abnormal ovary development MP:0003582; leave only under abnormal ovulation cycle MP:0009344
